traefik.sample.toml

```diff
@@ -996,43 +996,43 @@
 #
 # Optional
 #
-#[rancher]
```

Comment:
I think we want to leave all ranger configuration parameters commented out, as with the other providers in the sample file.

AFAIU, the idea is that users copy the file and uncomment just what they need.

See also #1455.